src/pages/Schedule/Calender/Date/Class/Class.js

```diff
@@ -1,12 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import * as S from '../../../StyledSchedule';
+import Modal from '../../../../../components/Modal/Modal';
+
+const Class = ({ classList }) => {
+  const [showModal, setShowModal] = useState(false);
+
+  const toggleModal = () => {
+    setShowModal(showModal => !showModal);
+  };
```

Comment:
```suggestion
  const toggleModal = () => {
    setShowModal(prev => !prev);
  };
```
작성하신 방법과 같은 동작을 하지만 좀 더 안전한 방법입니다.

Reply:
아! 감사합니다